sonarts-core/tests/rules/noDuplicateInCompositeRule/noDuplicateInCompositeRule.lint.ts

```diff
@@ -0,0 +1,23 @@
+type nokUnionDuplicate = number | number | string;
+//                                ^^^^^^ {{Remove this duplicated type.}}
+
+type nokIntersectionDuplicate = Person & Person & Loggable<Person>;
+//                                       ^^^^^^ {{Remove this duplicated type.}}
+
+type nokUnionDuplicate2 = number | number | number | string;
```

Comment:
@andrea-guarino-sonarsource add assertion for secondary location
1st appearance - secondary with message `Original`
2nd appear - primary
3rd and further - secondary `Another duplicate`